zeebe/dynamic-config/src/main/resources/proto/topology.proto

```diff
@@ -156,6 +158,16 @@ message StartPartitionScaleUpOperation {
   int32 desiredPartitionCount = 2;
 }
 
+message AwaitRedistributionCompletion{
+  int32 desiredPartitionCount = 1;
+  repeated int32 redistributedPartitions= 2;
```

Comment:
:wrench: I find the naming here a bit confusing. It seems to hold a set of partitions that require redistribution.
1. Do we need this at all? Isn't this implicitly known via `desiredPartitionCount`?
2. If we need it, shouldn't we call it `partitionsRequiringRedistribution` or something similar? This renaming should then trickly down into various APIs.

Reply:
The idea was that the broker is able to provide the partitions that have completed redistribution (i.e. `redistributedPartitions`) so that it's a bit more clear from the logs the progress of the operations.
When the expected paritions that need to be redistributed matches the list from the response, then the operation is considered completed

Reply:
That's a nice idea but it has two problems:
1. Operations are currently immutable, they cannot be changed to indicate progress
2. As of now, the field actually holds partitions that _have not_ completed redistribution, at least initially.

Reply:
Yeah sorry got confused :-) 
This is the operation that will be used to check whether the response form the broker (`GetScaleUpProgress`) contains all the required partitions. 
I'll change the name of the field :+1: